documentation/docs/welcome.md

```diff
@@ -18,25 +18,24 @@ keywords:
 
 # Welcome to INX-POI
 
-INX-POI enables you to generate and verify Proof-of-Inclusion of blocks in the Tangle.
-Given a piece of data or transaction and the proof, you can verify whether it was included in the Tangle at any given time.
+INX-POI generates and verifies proofs of inclusion of blocks. By generating a proof, you can always verify that the block used to be part of the Tangle. The Tangle nodes truncate old blocks by necessity, but with proofs of inclusion you can preserve important blocks on your own.
 
 ## Setup
```

Comment:
```suggestion
## Set Up
```

Reply:
I'd keep it as a node to keep it in line with the rest of the sections (configuration, API, source code)